src/components/eip4626/user/Rebalancer.tsx

```diff
@@ -0,0 +1,237 @@
+import {
+  Flex,
+  Button,
+  Heading,
+  Input,
+  FormControl,
+  FormHelperText,
+  Checkbox,
+} from "@chakra-ui/react";
+import { MenuSelect } from "@/components/MenuSelect";
+import { RebalancerModal } from "./RebalancerModal";
+import { useState } from "react";
+import { useTokenRebalancer } from "@/hooks/eip4626/mutations/useTokenRebalancer";
+import { GroupBase } from "react-select";
+import { EvmAddress } from "@/types/types";
+import { TransactionResult } from "@/components/TransactionResult";
+
+const pythPriceIds = [
+  "0x1111111111111111111111111111111111111111111111111111111111111111",
+  "0x2222222222222222222222222222222222222222222222222222222222222222",
+  "0xc2d94679a866c5411f85a572c82827b2f0ae880f9ca74ca5e16b070838d51328",
+];
+
+const vaultATokens = [
+  "0x4f9F90BdA689001dC21707e4f9F0FC803c03d701",
+  "0x6684B43CE4B22e5DAc9a40b11938f4C710C14aF4",
+  "0xe3F9ece8B56b8e3b4DECD9f101954Ee1D32159A7",
+];
+
+export const Rebalancer = () => {
+  const [isModalsOpen, setIsModalsOpen] = useState({
```

Comment:
From the variable name itself, I expected a simple t/f boolean. I would suggest to have 3 separate variables to lower the code coupling